src/Drupal/V8/DkanCommands.php

```diff
@@ -27,8 +27,23 @@ public function dkanTestCypress() {
 
     /**
      * Run DKAN PhpUnit Tests.
+     * 
+     * Because of a limitation of using the single `$args` array to pass all
+     * additional arguments, the phpunit specific arguments can be only be
+     * passed without the `--` prefix to avoid conflicting with Robo\Tasks
+     * built in arguments.
+     * 
+     * e.g. dtlk dkan:test-phpunit debug verbose coverage-html=../logs/coverage
+     *
+     * Note that only full option names will work and not single letter arguments
+     * i.e. use `verbose` instead of `v`
+     * 
+     * @todo currently `--testsuite` is a hardoced argument. Could refactor if additional test suites are added
```

Comment:
Let's keep @todos focused on things that we want to do right now but for whatever reason we didn't. This todo has a trigger in the future that might never happen. Let's remove it for now.

Reply:
fixed in c97157faa449424fde40458ef0a61e08a4391d40